RGBLED/CS/MainPage.xaml.cs

```diff
@@ -0,0 +1,208 @@
+            bluepin.Write(GpioPinValue.High);
+            bluepin.SetDriveMode(GpioPinDriveMode.Output);
+
+            GpioStatus.Text = string.Format(
+                "Red Pin = {0}, Green Pin = {1}, Blue Pin = {2}",
+                redpin.PinNumber,
+                greenpin.PinNumber,
+                bluepin.PinNumber);
+
+            timer = new DispatcherTimer();
+            timer.Interval = TimeSpan.FromMilliseconds(500);
+            timer.Tick += Timer_Tick;
+            timer.Start();
+        }
+
+        private void FlipLED()
+        {
+            Debug.Assert(redpin != null && bluepin != null && greenpin != null);
+
+            switch (ledStatus)
+            {
+                case LedStatus.Red:
+                    //turn on red
+                    redpin.Write(GpioPinValue.High);
+                    bluepin.Write(GpioPinValue.Low);
+                    greenpin.Write(GpioPinValue.Low);
+
+                    LED.Fill = redBrush;
+                    ledStatus = LedStatus.Green;    // go to next state
+                    break;
+                case LedStatus.Green:
+
+                    //turn on green
+                    redpin.Write(GpioPinValue.Low);
+                    greenpin.Write(GpioPinValue.High);
+                    bluepin.Write(GpioPinValue.Low);
+
+                    LED.Fill = greenBrush;
+                    ledStatus = LedStatus.Blue;     // go to next state
+                    break;
+                case LedStatus.Blue:
+                    //turn on blue
+                    redpin.Write(GpioPinValue.Low);
+                    greenpin.Write(GpioPinValue.Low);
+                    bluepin.Write(GpioPinValue.High);
+
+                    LED.Fill = blueBrush;
+                    ledStatus = LedStatus.Red;      // go to next state
+                    break;
+            }
+        }
+
+        private void Timer_Tick(object sender, object e)
+        {
+            FlipLED();
+        }
+
+        private void TurnOffLED()
+        {
+            redpin.Write(GpioPinValue.Low);
+            greenpin.Write(GpioPinValue.Low);
+            bluepin.Write(GpioPinValue.Low);
+        }
+
+        private void Delay_ValueChanged(object sender, RangeBaseValueChangedEventArgs e)
+        {            
+            if (timer == null)
+            {
+                return;
+            }
+
+            if (e.NewValue == Delay.Minimum)
+            {
+                DelayText.Text = "Stopped";
+                timer.Stop();
+                TurnOffLED();
+            }
+            else
+            {
+                DelayText.Text = e.NewValue + "ms";
+                timer.Interval = TimeSpan.FromMilliseconds(e.NewValue);
+                timer.Start();
+            }
+        }
+
+        public enum DeviceModel { RaspberryPi2, MinnowBoardMax, DragonBoard410, Unknown };
+
+        static DeviceModel GetDeviceModel()
+        {
+            var deviceInfo = new EasClientDeviceInformation();

```

Comment:
These list devices we don't support for IoT Enterprise.  Where is UpBoard?

Reply:
GetDeviceModel() is commented in the MainPage.xml.cs file. It is for raspberry pi and dragon board.

Upboard uses default setting.